contracts/sol6/SanityRatesGasPrice.sol

```diff
@@ -0,0 +1,99 @@
+pragma solidity 0.6.6;
+
+import "./IKyberSanity.sol";
+import "./utils/Utils5.sol";
+import "./utils/Withdrawable3.sol";
+
+/**
+ *   @title SanityRatesGasPrice contract
+ *   The contract is an update from the previous SanityRates contract, and provides
+ *   the following functionality:
+ *       - setting reasonable diff
+ *       - setting max gas price criteria for a trade
+ *       - setting sanity rates
+ *       - getting sanity rates
+ *
+ *       What's new from previous SanityRates contract:
+ *           1) Setting max gas price - essentially allows the reserve manager to
+ *              limit taker's trade gas price, in order to discourage front running
+ *              price updates.
+ *           2) Fix to https://github.com/KyberNetwork/smart-contracts/issues/168.
+ *           3) Update to Solidity 6.
+ */
+
+contract SanityRatesGasPrice is IKyberSanity, Withdrawable3, Utils5 {
+    struct SanityData {
+        uint168 tokenRate;
```

Comment:
lets use here:
 uint128
 uint128

those numbers look a little weird.
especially when later used for casting in the file.

Reply:
Done.